pywbem/mof_compiler.py

```diff
@@ -2178,6 +2191,41 @@ def CreateClass(self, *args, **kwargs):
             self.classes[self.default_namespace] = \
                 NocaseDict({cc.classname: cc})
 
+        objects = list(cc.properties.values())
+        for meth in cc.methods.values():
+            objects += list(meth.parameters.values())
+
+        for obj in objects:
+            # Validate that reference_class exists in repo
+            if obj.type == 'reference':
+                try:
+                    self.GetClass(obj.reference_class, LocalOnly=True,
+                                  IncludeQualifiers=True)
+                except CIMError as ce:
+                    if ce.args[0] == CIM_ERR_NOT_FOUND:
```

Comment:
Please access the CIM status via the `cim_status` attribute of the `CIMError` object.

Reply:
Create issue to fix the ce.args.